### PR TITLE
Add configuration params description readme

### PR DESCRIPTION
**Pull request by hqkate**
添加了配置文件参数说明。
中文版： configs/参数配置文档说明.md
英文版：configs/config_description.md

Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [x] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: configs/参数配置文档说明.md
Comment (SamitHuang) [inline configs/参数配置文档说明.md:1]: 表格中，“例子” -> "默认值"，如无默认值 ，填 -

Comment (hqkate) [reply configs/参数配置文档说明.md:1]: 已更新~

**Review by HaoyangLee — APPROVED** [review]

File: configs/参数配置文档说明.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+## 6. 损失函数 (loss)
+
+代码位置请看： [mindocr/losses](../mindocr/losses)
+
+| 字段 | 用途 | 默认值 | 备注 |
+| :---- | :---- | :---- | :---- |
+| name | 损失函数类名 | - | 目前支持 CTCLoss, L1BalancedCELoss |
+| pred_seq_len | 预测文本的长度 | 26 | 由网络架构决定 |
+| max_label_len | 最长标签长度 | 25 | 数值应小于网络预测文本的长度 |
+| batch_size | 单卡批量大小 | 32 | \ |
```
Comment (SamitHuang) [inline configs/参数配置文档说明.md:93]: 批量大小？ 批大小

File: configs/参数配置文档说明.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+代码位置请看： [mindocr/scheduler](../mindocr/scheduler)
+
+| 字段 | 用途 | 默认值 | 备注 |
+| :---- | :---- | :---- | :---- |
+| scheduler | 学习率调整策略名 | 'constant' | 目前支持 'constant', 'cosine_decay', 'step_decay', 'exponential_decay', 'polynomial_decay', 'multi_step_decay' |
+| min_lr | 学习率最小值 | 1e-6 | \ |
+| lr | 学习率峰值 | 0.01 | \ |
+| num_epochs | 最大训练epoch数 | 200 | \ |
+| warmup_epochs | 学习率递增epoch数 | 3 | \ |
```
Comment (SamitHuang) [inline configs/参数配置文档说明.md:108]: 描述有点改进

File: configs/参数配置文档说明.md
Comment (SamitHuang) [inline configs/参数配置文档说明.md:1]: 参数说明描述过于简洁，对beginner不友好，可以详细些。

**Review by zhtmike — APPROVED** [review]

**Pull request merged**